ReactiveCocoa.playground/Pages/Signal.xcplaygroundpage/Contents.swift

```diff
@@ -1,11 +1,15 @@
 /*:
  > # IMPORTANT: To use `ReactiveCocoa.playground`, please:
  
- 1. Open `ReactiveCocoa.xcworkspace`
```

Comment:
Oh, now I understand the Problem in #2916. I was confused because the Playground instructions in the [Readme](https://github.com/ReactiveCocoa/ReactiveCocoa#playground) already state that you need to run `script/bootstrap`. 
Maybe you could also update the instructions in these instructions in the Readme